### ISSUE TITLE
downgrading to 0.44 from 0.45 fails

### ISSUE THREAD

**Issue by jmls** [RocketChat/Rocket.Chat#4809]
after experiencing audio problems with 0.45 (https://github.com/RocketChat/Rocket.Chat/issues/4808) we tried to downgrade to 0.44

Unfortunately, the service does not start , and this is in the logs:

```
11/1/2016 1:02:27 PMUpdating process.env.MAIL_URL
11/1/2016 1:02:28 PMufs: permissions are not defined for store "fileSystem"
11/1/2016 1:02:28 PMufs: store created at
11/1/2016 1:03:43 PMUpdating process.env.MAIL_URL
11/1/2016 1:03:43 PMufs: permissions are not defined for store "fileSystem"
11/1/2016 1:03:43 PMufs: store created at
11/1/2016 1:03:54 PM[33mNot migrating, control is locked. Attempt 1/30. Trying again in 10 seconds.[39m
11/1/2016 1:04:04 PM[33mNot migrating, control is locked. Attempt 2/30. Trying again in 10 seconds.[39m
11/1/2016 1:04:14 PM[33mNot migrating, control is locked. Attempt 3/30. Trying again in 10 seconds.[39m
11/1/2016 1:04:24 PM[33mNot migrating, control is locked. Attempt 4/30. Trying again in 10 seconds.[39m
11/1/2016 1:04:34 PM[33mNot migrating, control is locked. Attempt 5/30. Trying again in 10 seconds.[39m
11/1/2016 1:04:44 PM[33mNot migrating, control is locked. Attempt 6/30. Trying again in 10 seconds.[39m
11/1/2016 1:04:54 PM[33mNot migrating, control is locked. Attempt 7/30. Trying again in 10 seconds.[39m
```

Is it possible to downgrade ?

**Comment by MartinSchoeler** [comment]
@jmls Unfortunately we don't support downgrades

**Comment by engelgabriel** [comment]
@jmls you have to access mongoDB directly, open the `migrations` collection and change the `version` of the control record to `63`:

```
{
  "_id": "control",
  "version": 63,
  "locked": false
}
```

**Comment by engelgabriel** [comment]
Please reopen if this manual process of downgrading didn't work.

**Comment by asthomasdk** [comment]
Thanks Gabriel! 

I think we will stick with the 0.45.0 version we have right now - and wait for the audio to be fixed. We are able to use the video call feature until then.

**Comment by prabinsubedi** [comment]
How to down grade the rocket chat version from 0.48.0 to 0.47.0. I am having problems with API call. 
Thanks Much.

**Comment by engelgabriel** [comment]
What API call?

The docs have been updated at
https://rocket.chat/docs/developer-guides/rest-api/

Gabriel Engel

**Comment by yanntech** [comment]
@prabinsubedi just rollbacked for same reason (webhook integration)
You just need to rollback 0.47.1 and update mongodb:

```
db.migrations.update({"version":71},{"version":69})
```

**Comment by engelgabriel** [comment]
webhook integration have been fixed on 0.48.1 that is being released just now.

**Comment by prabinsubedi** [comment]
Thanks everyone for prompt reply one it works now with 0.48.1.
